src/screens/home/hooks/useLauncherClient.ts

```diff
@@ -20,9 +20,14 @@ export const useLauncherClient = (
 
     const slug = launcherContextValue?.konnector.slug
```

Comment:
FYI this will fail if we somehow get an odd object named `connector` instead of `konnector`, IDK if it's worth preventing the issue. As of now there is no safety net and I had multiple crashes where this error happened

Reply:
I don't know how we can have a `connector` object. If you still got this issue, we should dig in to fix the root cause.